discord/webhook/async_.py

```diff
@@ -644,8 +644,9 @@ def handle_message_parameters(
         payload["embeds"] = [] if embed is None else [embed.to_dict()]
     if content is not MISSING:
         payload["content"] = str(content) if content is not None else None
+    attachments = []
```

Comment:
This makes the line below obsolete

Reply:
then += instead? it needs to be defined outside the if

Reply:
Or it needs to be put in an else after the if